src/amber/router/session/redis_store.cr

```diff
@@ -4,17 +4,17 @@ require "uuid"
 module Amber::Router::Session
   class RedisStore < AbstractStore
     @id : String?
-    getter store : Redis
+    getter store : Redis = Redis.new(url: Amber.settings.redis_url.to_s)
```

Comment:
Isn't it a better idea to specify this in the initialize? Maybe even with a check if redis_url is specified. If it's not used than raise an exception stating so?